doc/packaging.md

```diff
@@ -231,9 +231,9 @@ We need a recent version of Abseil.
 
 ```bash
```

Comment:
We are missing `Abseil` as a dependency further up in this file.  We probably want to update the minimum version for gRPC too.  I doubt we still compile with 1.16

Reply:
Ack. Thanks. I filed https://github.com/googleapis/google-cloud-cpp/issues/5796 to track this.